arch/Kconfig

```diff
@@ -837,6 +837,14 @@ config ARCH_STACKDUMP
 	---help---
 		Enable to do stack dumps after assertions
 
+if ARCH_STACKDUMP
+
+config ARCH_STACKDUMP_MAX_LENGTH
+	int "The maximum length for dump stack on assertions"
```

Comment:
depends on ARCH_STACKDUMP and remove if/endif

Reply:
Done.